pkg/installmanager/installmanager.go

```diff
@@ -731,22 +741,6 @@ func (m *InstallManager) gatherBootstrapNodeLogs(cd *hivev1.ClusterDeployment) e
 	return nil
 }
 
-// chmodSSHPrivateKey copies the mounted volume with the ssh private key to
```

Comment:
Why are we losing this functionality from a recent PR?

Reply:
Great catch, looks like a merge issue.

Reply:
This code still needs to be restored, right?

Reply:
Sorry I have no idea how I lost this, I remember manually restoring it.

Reply:
Oh boy, I see what happened, I'm squashing it into a commit prior to where I removed it. It squashes clean and then the later commit removes it again.